packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -49,6 +49,14 @@ module.exports = (themeOptions = {}) => {
             node { etc...
       */
 
+      // Color presets
+      {
+        resolve: 'gatsby-source-filesystem',
+        options: {
+          name: 'colorPresets',
+          path: path.join(__dirname, `./color-presets`),
```

Comment:
to be on the safe side, I would ignore the readme and code to prevent them from being handled as content by other plugins that don't explicitly filter out the name of the source plugin (which most don't)

```suggestion
          path: path.join(__dirname, `./color-presets`),
          ignore: [`**/*.md`,`**/*.js`], 
```
PSEUDOCODE!

Reply:
Good point 👌